client/packages/coldchain/src/Equipment/DetailView/UpdateStatusButton.tsx

```diff
@@ -43,10 +45,19 @@ export const UpdateStatusButtonComponent = ({
   const { currentTab, onChangeTab } = useTabs(Tabs.Status);
   const t = useTranslation();
   const { Modal, hideDialog, showDialog } = useDialog({ onClose });
-  const { error, success } = useNotification();
+  const { error, success, info } = useNotification();
   const [draft, setDraft] = useState<Partial<Draft>>(getEmptyAssetLog(''));
   const { insertLog, invalidateQueries } = useAssets.log.insert();
 
+  const permission = UserPermission.AssetMutate;
+  const { userHasPermission } = useAuthContext();
+
+  const onUpdateStatus = () => {
+    if (userHasPermission(permission)) {
+      showDialog;
```

Comment:
Need to call this - the modal isn't opening 😁

Reply:
Done, thanks!